requirements/dev.in

```diff
@@ -8,22 +8,21 @@ freezegun==1.5.2
 markdown-it-py>=2.2.0
 pipdeptree==2.26.1
 py==1.11.0
-Pygments>=2.15.0  # to bring it up to a secure version
 PyPOM==2.2.4
 pyquery==2.0.1
-pytest==8.4.0
-pytest-cov==6.1.1
+pytest==8.4.1
+pytest-cov==6.2.1
 pytest-datadir==1.7.2
 pytest-django==4.11.1
 pytest-mock==3.14.1
 pytest-parallel==0.1.1
 pytest-rerunfailures==15.1
-pytest-selenium==4.0.1
+pytest-selenium==4.1.0
 responses==0.25.7
 ruff==0.11.13
-selenium==4.9.1  # Pinned to 4.9.1 until https://github.com/pytest-dev/pytest-selenium/issues/315 is resolved
+selenium==4.34.2
 translate-toolkit==3.15.2
-uv==0.7.12
+uv==0.7.20
```

Comment:
Also to be on the safe side, to a) have the latest version actually resolving the deps, b) match what is currently used in the compile-requirements container to actually freeze them.